app/models/opportunity.rb

```diff
@@ -216,11 +216,11 @@ def confidential?
   # remove voucher
   # remove self (opportunity)
 
-  def stop_matches_and_remove_entire_history_from_existance!
+  def stop_matches_and_remove_entire_history_from_existence!
```

Comment:
This had a typo, but as far as I could tell isn't used anywhere, probably a manual mechanism for data cleanup?